### ISSUE TITLE
When the try dyson account login, error "Unknown error occurred"

### ISSUE THREAD

**Issue by lux150** [libdyson-wg/ha-dyson#155]
Hello,

I tried to log in with my official Dyson account.
But I saw the error message as below.
`Unknown error occurred`

- Add-on Version 1.3.11

Have you experienced any recent problems with the ability to log in directly to your account and bring up entities?
I'm curious about the solution.

Thank you :)

**Comment by lux150** [comment]
I solved the problem first in the following way.
- App Version Downgrade (1.3.11 → 1.3.7)
- Add a device after searching for a product
- Update app after normal login verification (1.3.7 → 1.3.11)
- Add a device after searching for a product

However, it seems that the entities of all products are not imported.
Only the On/Off switch has been added as a component, and it seems that the fan speed and other information cannot be accurately imported.

When you click on the entity, the following functions were possible on the details screen.
- Change preset mode (Auto & Normal)
- Change of direction (Forward & Reverse)
- Rotation settings (Yes & No)

My device is **TP-07** and when registering the device, the local IP address was also set up well.
